src/test/ui/issue-48636.stderr

```diff
@@ -0,0 +1,24 @@
+error[E0585]: found a documentation comment that doesn't document anything
+  --> $DIR/issue-48636.rs:13:5
+   |
+LL |     /// The id of the parent core
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = help: doc comments must come before what they document, maybe a comment was intended with `//`?
+
+error: expected `,`, or `}`, found `/// The id of the parent core`
+  --> $DIR/issue-48636.rs:13:5
+   |
+LL |     /// The id of the parent core
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = help: struct fields should be separated by commas
```

Comment:
I feel that in this case the desired full output would be:

```
error[E0585]: found a documentation comment that doesn't document anything
  --> $DIR/issue-48636.rs:13:5
   |
LL |     x: u8
   |          - help: missing comma here: `,`
LL |     /// The id of the parent core
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: doc comments must come before what they document, maybe a comment was intended with `//`?
```

using a `span_suggestion`.

Reply:
done!